customization/windows-template-customization.md

```diff
@@ -127,23 +127,19 @@ Manual tasks that can/should be started in the template
 
  * Windows Update
 
- * Windows file cleaning
-    1. Run windows drive cleaner as Administrator.
+ * Windows Disk Cleanup
+    1. Run Windows Disk Cleanup as Administrator.
     2. Enable all the task and run the cleaner
 
- * CCleaner file cleaning
-    1. Install CCleaner free
-    2. Copy the attached ccleaner configuration file in CCleaner program file folder
-    3. Run ccleaner with all option set except "wipe free space" (it will also remove user history and preferences)
-    4. Run ccleaner only with the option "wipe free space".
-
-        It will write zeros in all unused space. This will allow you to strip the root.img file later
```

Comment:
You can reword these steps to be more generic if you want, but do not remove them. This information is necessary.